tmt/export/__init__.py

```diff
@@ -40,7 +40,11 @@
 if TYPE_CHECKING:
     import tmt.base
 
-TEMPLATES_DIRECTORY = Path(pkg_resources.resource_filename('tmt', 'export/templates'))
+TEMPLATES_DIRECTORY = (Path(importlib.resources.files('tmt'))  # type: ignore[arg-type]
+                       / 'export/templates')
+# ignore[arg-type]: importlib.resources.files() returns 'Traversable' object,
+# though in our use-case it should always produce a pathlib.PosixPath object.
+# Converting it to tmt.utils.Path object should not cause any issues regardless.
```

Comment:
@happz is used to place the explanatory comments about the command, let's keep consistency here.

Reply:
Presumably this would be in the docstring in the helper function. 
I suck at this - is this ok?

    """Helper function to get path of package files or directories.

    Example: 
        resource_files('schemas') -> Path('/usr/lib/python3.11/site-packages/tmt/schemas')
    """

Reply:
No worries. I'd suggest the following:

```
"""
Helper function to get path of package file or directory.

A thin wrapper for :py:func:`importlib.resources.files`: ``files()`` returns ``Traversable`` object,
though in our use-case it should always produce a :py:class:`pathlib.PosixPath` object.
Converting it to :py:class:`tmt.utils.Path` instance should be safe and stick to the "``Path`` only!"
rule in tmt's code base.

:param path: file or directory path to retrieve, relative to the ``package`` root.
:param package: package in which to search for the file/directory.
:returns: an absolute path to the requested file or directory.
"""
```

Reply:
Awesome. Thanks a lot.